src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,136 @@
+package duke;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.Todo;
+
+import java.util.*;
+import java.lang.*;
+
+public class Duke {
+    public static void main(String[] args) throws IndexOutOfBoundsException {
+
+        Scanner sc = new Scanner(System.in);
+        DukeException ex = new DukeException();
+
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+
+        ArrayList<Task> tasks = new ArrayList<Task>();
+
+        System.out.println("Hello from\n" + logo);
+        System.out.println("Hello! I'm Duke.Duke\n" + "What can I do for you?");
+        displayOptions();
+        String userInput;
+        do {
+            userInput = sc.nextLine();
+            if(userInput.equals("list")){
+                System.out.println("-------------------------------------------------------------------------\n" + "Here are the tasks in your list:");
+
+                int i=1;
+                for(Task task: tasks){
+                    System.out.println(i + ": " + task.getDescription());
+                    i++;
+                }
+                i=0;
```

Comment:
Avoid magic numbers. If the value is an iterator, consider putting into a `for` loop instead.